packages/amplify-e2e-tests/src/__tests__/auth_6.test.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  initJSProjectWithProfile,
+  getProjectMeta,
+  deleteProject,
+  amplifyPushAuth,
+  createNewProjectDir,
+  deleteProjectDir,
+  addAuthWithMaxOptions,
+} from 'amplify-e2e-core';
```

Comment:
nit: organize imports. In VSCode, right-click => Source Action => Organize Imports

Reply:
updated